canvaplay/src/Pages/Watchlater/Watchlater.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
you can exclude this as we dont need to import react in every file.

Reply:
Sure. Thanks